examples/state_containers_examples/public/todo.tsx

```diff
@@ -0,0 +1,327 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useEffect } from 'react';
+import { Link, Route, Router, Switch, useLocation } from 'react-router-dom';
+import { History } from 'history';
+import {
+  EuiButton,
+  EuiCheckbox,
+  EuiFieldText,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentBody,
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiTitle,
+} from '@elastic/eui';
+import {
+  BaseStateContainer,
+  createKbnUrlSyncStrategy,
+  createStateContainer,
+  createStateContainerReactHelpers,
+  PureTransition,
+  INullableBaseStateContainer,
+  syncStates,
+  useUrlTracker,
+} from '../../../src/plugins/kibana_utils/public';
+import {
+  defaultState,
+  pureTransitions,
+  TodoActions,
+  TodoState,
+} from '../../../src/plugins/kibana_utils/demos/state_containers/todomvc';
+import { createSessionStorageSyncStrategy } from '../../../src/plugins/kibana_utils/public/state_sync/state_sync_strategies';
+import { getStateFromKbnUrl } from '../../../src/plugins/kibana_utils/public/state_management/url';
```

Comment:
I think this stuff should be exported from a top level index.ts file in this plugin.  I'm surprised this doesn't give an eslint error for importing from a restricted zone.

Reply:
My guess is it does not error because the file is `/examples/*` not `/src/plugins/*`.

Reply:
Thanks, fixed those imports

Reply:
@stacey-gammon, @streamich, this should fix the rule: https://github.com/elastic/kibana/pull/54252